net.sapples.LiveCaptions.json

```diff
@@ -0,0 +1,108 @@
+{
+    "app-id" : "net.sapples.LiveCaptions",
```

Comment:
the domain `sapples.net` doesn't answer, and there is no reference anywhere to it beside here. Which put in question whether it meets the requirements.

Reply:
I haven't set up a proper home page yet but april.sapples.net and www.sapples.net should respond